youtube_dl/extractor/piksel.py

```diff
@@ -16,18 +16,32 @@
 
 class PikselIE(InfoExtractor):
     _VALID_URL = r'https?://player\.piksel\.com/v/(?P<id>[a-z0-9]+)'
-    _TEST = {
-        'url': 'http://player.piksel.com/v/nv60p12f',
-        'md5': 'd9c17bbe9c3386344f9cfd32fad8d235',
-        'info_dict': {
-            'id': 'nv60p12f',
-            'ext': 'mp4',
-            'title': 'فن الحياة  - الحلقة 1',
-            'description': 'احدث برامج الداعية الاسلامي " مصطفي حسني " فى رمضان 2016علي النهار نور',
-            'timestamp': 1465231790,
-            'upload_date': '20160606',
+    _TESTS = [
+        {
+            'url': 'http://player.piksel.com/v/nv60p12f',
+            'md5': 'd9c17bbe9c3386344f9cfd32fad8d235',
+            'info_dict': {
+                'id': 'nv60p12f',
+                'ext': 'mp4',
+                'title': 'فن الحياة  - الحلقة 1',
+                'description': 'احدث برامج الداعية الاسلامي " مصطفي حسني " فى رمضان 2016علي النهار نور',
+                'timestamp': 1465231790,
+                'upload_date': '20160606',
+            }
+        },
+        {
+            'url': 'http://www.uscourts.gov/cameras-courts/state-washington-vs-donald-j-trump-et-al',
```

Comment:
All tests placed here must have URLs that match extractor's `_VALID_URL`. Tests for generic embeds should be placed into generic extractor.
```
PS C:\Dev\youtube-dl\master> py -3.5 .\test\test_all_urls.py
...F.............
======================================================================
FAIL: test_no_duplicates (__main__.TestAllURLsMatching)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".\test\test_all_urls.py", line 94, in test_no_duplicates
    self.assertTrue(ie.suitable(url), '%s should match URL %r' % (type(ie).__name__, url))
AssertionError: False is not true : PikselIE should match URL 'http://www.uscourts.gov/cameras-courts/state-washington-vs-donald-j-trump-et-al'

----------------------------------------------------------------------
Ran 17 tests in 8.066s

FAILED (failures=1)
```